app/components/primer/progress_bar_component.rb

```diff
@@ -52,13 +52,14 @@ class Item < ViewComponent::Slot
       include ClassNameHelper
       attr_reader :kwargs
 
-      def initialize(percentage: 0, bg: :green, **kwargs)
+      def initialize(percentage: 0, bg: :green, style: "", **kwargs)
         @percentage = percentage
         @kwargs = kwargs
+        @style = style
 
         @kwargs[:tag] = :span
         @kwargs[:bg] = bg
-        @kwargs[:style] = "width: #{@percentage}%;"
+        @kwargs[:style] = @style + "width: #{@percentage}%;"
```

Comment:
If someone forgets to include a terminating semicolon when passing in `style`, this will break subtly. What if we instead added the provided `style` to the end?

```suggestion
        @kwargs[:style] = "width: #{@percentage}%; #{@kwargs[:style]}"
```

Reply:
good catch!